.circleci/config.yml

```diff
@@ -57,13 +61,7 @@ jobs:
 
       - run:
           name: deploy
-          command: |
-            echo "staging deploy"
-            pwd
-            cd ..
-            cd repo
-            echo '---------'
-            ./node_modules/serverless/bin/serverless
+          command: yarn && yarn build && yarn --production=true && sls deploy -s staging
```

Comment:
Wondering if you need the `yarn && yarn build` as they've already run? I guess by doing this though we don't have to worry about re-ordering things in future or always having the workspace persisted so probably more flexible this way for the sake of 20 seconds or so.

Reply:
hmm good point, i will take it out